Makefile

```diff
@@ -66,6 +66,11 @@ mock_gen:
 
 .PHONY: test
 test:
+	go test -mod=vendor -race ./...
+	cd konnectivity-client && go test -race ./...
+
+.PHONY: test-coverage
+test-coverage: test
```

Comment:
A little concerned we have lowered what `make test` covers?

Reply:
per discussion I will keep `test` as-is, and split out a `fast-test`.